source/_integrations/switch.command_line.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Command line Switch"
+title: "Command Line Switch"
```

Comment:
The title is synced from the code base, changing it here will be reverted automatically.